completions.go

```diff
@@ -27,6 +27,30 @@ type CreateCompletionsRequest struct {
 	User             string            `json:"user,omitempty"`
 }
 
+type CreateCompletionsResponse struct {
+	ID      string  `json:"id,omitempty"`
+	Object  string  `json:"object,omitempty"`
+	Created int     `json:"created,omitempty"`
+	Model   string  `json:"model,omitempty"`
+	Choices Choices `json:"choices,omitempty"`
+	Usage   Usage   `json:"usage,omitempty"`
+	Error   Error   `json:"error,omitempty"`
+}
+
+type Usage struct {
+	PromptTokens     int `json:"prompt_tokens,omitempty"`
+	CompletionTokens int `json:"completion_tokens,omitempty"`
+	TotalTokens      int `json:"total_tokens,omitempty"`
+}
+
+type Choices []struct {
```

Comment:
The choices field is not the same in all the structs, it only contains the same name, so I would avoid share structures